server/app/boot_jobs.rb

```diff
@@ -5,6 +5,5 @@
 unless ENV['RACK_ENV'] == 'test' || ENV['NO_MONGO_PUBSUB']
```

Comment:
Turns out that the `bin/kontena-console` and `rake kontena:reset_admin` processes do not even load `app/boot_jobs.rb`, so AFAIK the `NO_MONGO_PUBSUB` env does not actually have any effect on anything.

OTOH it does mean that if something else is changed and those processes do end up requiring `app/boot_jobs.rb`, then they would still be protected from launching the supervisors/workers. IMO for this purpose it would actually be better to `fail if ENV['NO_MONGO_PUBSUB']` instead?